maven-plugin/src/main/java/org/openmrs/maven/plugins/Setup.java

```diff
@@ -186,6 +195,21 @@ private DistroProperties resolveDistroProperties(Server server) throws MojoExecu
 					}
 					platformMode = false;
 					break;
+				case O3_REFERENCE_APPLICATION:
+					String distro = generateO3ReferenceDistro();
```

Comment:
Does this ignore one supplied as an argument for distro.properties by advanced users?

Reply:
Yes. If the user supplies a  distro.properties they won't get the prompt
```
You can setup the following servers:
1) Distribution
2) Platform
3) O3 Reference Application

Which one do you choose? [1/2/3]: 
```